.env

```diff
@@ -3,6 +3,9 @@ INFERNO_HOST=http://localhost
 G10_VALIDATOR_URL=http://validator_service:4567
 REDIS_URL=redis://redis:6379/0
 
+# Full path to a custom JWKS json file, will use lib/onc_certification_g10_test_kit/bulk_data_jwks.json if left blank
+G10_BULK_DATA_JWKS=
```

Comment:
```suggestion
# G10_BULK_DATA_JWKS=
```

Reply:
oh you meant for it to be commented it, I thought you wanted it to be set. got it